search/docker-compose.yml

```diff
@@ -46,7 +46,8 @@ services:
       - repository_dir:/code/repository
       - upload_dir:/code/upload
       - download_dir:/code/download
-    command: python manage.py runserver 0.0.0.0:8000
+      - mystatic_dir:/code/mystatic
+    command: bash -c "python manage.py makemigrations && python manage.py migrate && python manage.py collectstatic <<<yes && gunicorn core.wsgi -b 0.0.0.0:8000"
```

Comment:
```suggestion
    command: bash -c "python manage.py makemigrations && python manage.py migrate && python manage.py collectstatic --noinput && gunicorn core.wsgi -b 0.0.0.0:8000"
```

Reply:
It might be asking whether to overwrite existing files.